torch/autograd/forward_ad.py

```diff
@@ -75,8 +75,8 @@ def make_dual(tensor, tangent, *, level=None):
 def unpack_dual(tensor, *, level=None):
     r"""Unpacks a "dual object" to return a namedtuple ``(primal, tangent)`` where
```

Comment:
This renders weird on the short version.
Would be nice to have something closer to the make_dual function.

In particular, we can split this into two sentences, the first one without explicit link to the attributes:
Unpacks a "dual object" to get both the Tensor value and its tangent (the forward AD gradient). The result is a named tuple ``(primal, tangent)`` ...

How does that render?

Reply:
I tweaked the wording a bit. Also updated `make_dual` to match:
![image](https://user-images.githubusercontent.com/13428986/150889949-f291ec87-8d65-45b9-84f4-7793298f1d36.png)